entity-framework/core/miscellaneous/cli/dbcontext-creation.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Design-time DbContext Creation - EF Core
+author: bricelam
+ms.author: bricelam
+ms.date: 10/27/2017
+ms.technology: entity-framework-core
+---
+Design-time DbContext Creation
+==============================
```

Comment:
Nit: We are using hashes for titles, at least in the majority of pages. Can you maintain the same style?

Reply:
I noticed there are several things that I prefer to do differently. Should we adopt guidelines? Would we push back on contributors who don't follow them?

Reply:
For background, I developed my style while working on my blog over the past few years. I prefer syntax that makes the raw markdown more readable/scannable (to my eyes).

Reply:
Can you file an issue so that we discuss it? I understand what you mean, I am just not familiar enough with how to do h3, h4, etc. using your style, and I think there is some benefit on having just consistency. 

I wouldn't block on this.